.github/workflows/deploy-site.yml

```diff
@@ -0,0 +1,60 @@
+name: Deploy page
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  build-faucet:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: client
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 18
+      - run: npm ci
+      - run: npm run check
+      - run: npm run build
+        env:
+          VITE_CAPTCHA_KEY: 6LftP5wkAAAAAH5cbuutkYKKaTnUMTLPqWlyeIL6
+          VITE_FAUCET_URL: https://ink-docs-rococo-faucet.parity-testnet.parity.io/drip
```

Comment:
this is temporary right?
if that's gonna be an alternative to current westend/rococo in matrix, there should be separate instances of server with its own wallets inside. So there will be 2-3 deploy-site.yml jobs, each with its own VITE_FAUCET_URL?

Reply:
nit: perhaps would be a bit more flexible going forward to have this as host, and /drip or /balance would be defined in the query/request methods

Reply:
Yes, I copy this from the implementation of !ink, but I need to sync with @rzadp so that we can see how we are gonna deploy it.

Also, this implementation uses captcha v3 (less invasive) while that endpoint uses captcha v2, so I think it also needs a slightly different code.